roxie/ccd/ccddebug.cpp

```diff
@@ -114,6 +114,22 @@ class InputProbe : public CInterface, implements IFinalRoxieInput, implements IE
         else
             return in->queryConcreteInput(idx);
     }
+    virtual IEngineRowStream *queryConcreteInputStream(unsigned idx)
```

Comment:
I think this is misnamed - it should really be queryConcreteOutputStream - it is the input to the class requesting it, but the output of the class being requested.  Any normal input/stream should return itself.  Only N-way oddities should do anything different.